frontend/src/features/rollout-announcement/components/NewFeatureContent.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useRef } from 'react'
+import { Box, BoxProps, ModalBody, ModalHeader, Text } from '@chakra-ui/react'
+import lottie from 'lottie-web'
+
+import { NewFeatureTag } from './NewFeatureTag'
+
+const TopSpacer = (): JSX.Element => (
+  <Box
+    width="100%"
+    height="4.5rem"
+    backgroundColor="primary.100"
+    borderTopRadius="1rem"
+  />
+)
+
+interface LottieAnimationProps extends BoxProps {
+  animationData: unknown
+}
+
+export const LottieAnimation = ({
+  animationData,
+  ...boxProps
+}: LottieAnimationProps) => {
+  const element = useRef<HTMLDivElement>(null)
+  const lottieInstance = useRef<unknown>()
+
+  useEffect(() => {
+    if (!element.current) return
+
+    lottieInstance.current = lottie.loadAnimation({
+      animationData,
+      container: element.current,
+    })
+
+    return () => {
+      lottieInstance.current = lottie.destroy()
+    }
+  }, [animationData])
+
+  return <Box {...boxProps} ref={element} />
+}
+
+interface NewFeatureContentProps {
+  title: string
+  description: string
+  animationData: unknown
+}
+
+export const NewFeatureContent = (props: {
+  content: NewFeatureContentProps
+}): JSX.Element => {
+  const { title, description, animationData } = props.content
+
+  return (
+    <>
+      <TopSpacer />
+      <LottieAnimation animationData={animationData} />
+      <ModalHeader>
+        <NewFeatureTag />
+        <Text paddingTop="0.5rem">{title}</Text>
```

Comment:
nit: use margins instead for spacing `mt`. The value required seems to be 0.625rem (10px) instead of 8.

Reply:
Okay updated back to `0.625rem`, I rounded off to nearest 0.25rem